### PR TITLE
xds: restructure XdsLoadBalancer Part 3: LookasideLb

### PR DESCRIPTION
**Pull request by dapengzhang0**

### CODE REVIEW

**Comment by ejona86** [comment]
Comment to see if easycla notices.

File: xds/src/test/java/io/grpc/xds/LookasideLbTest.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+import io.grpc.xds.XdsComms.AdsStreamCallback;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+/**
+ * Tests for {@link LookasideLb}.
+ */
+public class LookasideLbTest {
```
Comment (voidzcy) [inline xds/src/test/java/io/grpc/xds/LookasideLbTest.java]: `@RunWith(JUnit4.class)`

File: xds/src/test/java/io/grpc/xds/LookasideLbTest.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        .build();
+    lookasideLb.handleResolvedAddresses(resolvedAddresses);
+
+    verify(balancer1).handleResolvedAddresses(resolvedAddresses);
+
+    verify(balancer1, never()).shutdown();
+    assertThat(helpers).hasSize(1);
+    assertThat(balancers).hasSize(1);
+
+    // change fallback policy to fallback_2
```
Comment (voidzcy) [inline xds/src/test/java/io/grpc/xds/LookasideLbTest.java]: I guess this comment is wrong.

File: xds/src/test/java/io/grpc/xds/LookasideLbTest.java (hunk 1 above)
Comment (dapengzhang0) [reply xds/src/test/java/io/grpc/xds/LookasideLbTest.java]: Done.

File: xds/src/test/java/io/grpc/xds/LookasideLbTest.java (hunk 2 above)
Comment (dapengzhang0) [reply xds/src/test/java/io/grpc/xds/LookasideLbTest.java]: Changed. The test must have copied some other test as a template.

**Pull request merged**